### ISSUE TITLE
Splash Page 

### ISSUE THREAD

**Issue by smcintosh881** [moltimate/moltimate-frontend#22]
## Expected Behavior

Details of project
Development team
Github repo
Some project history
Funding 

## Context (Environment)

We need a landing page experience to let people know what we have to offer, and how this project is being funded

**Comment by smcintosh881** [comment]
@blackpan2  can you comment the funding information that our sponsors want present so I can make sure I don't miss anything?

**Comment by blackpan2** [comment]
Here is the information from the sponsors that they wanted on this page.
```
Developed by the SBEVSL Project
Partial funding provided by:
NSF DUE 0402408
NIGMS 1R15GM078077
NIGMS 1R15GM078077-01
NIGMS 3R15GM078077-01S
NIGMS 2R15GM078077-02
NIGMS 3R15GM078077-02S1
NSF IUSE 1503811
NSF IUSE 1709170
RIT
Dowling College
```
![basillogo](https://user-images.githubusercontent.com/11465135/49969997-055b1580-fef8-11e8-84dd-baa2c45d2a46.jpg)
![sbevsllogo](https://user-images.githubusercontent.com/11465135/49970000-068c4280-fef8-11e8-9bc7-2b4a032a422f.jpg)

**Comment by smcintosh881** [comment]
@blackpan2  can you email me higher res images?

**Comment by blackpan2** [comment]
@smcintosh881 Those are the logos the sponsors sent me I will copy you on an email to see if they have a higher resolution one.

Search Motif: 
> Query with a PDB entry to find its optimal alignment among a collection of motif templates. 

Create Motif:
> Create new motif templates to expand the options of your alignment searching and find more results.

**Comment by blackpan2** [comment]
For completeness this is being resolved by #23
